File: Reto # - 11/Java/Fichero.java
/**
 * @author DeathbatO
 */

import java.io.File;
import java.io.FileWriter;
import java.io.FileReader;
import java.io.BufferedReader;
import java.io.IOException;

public class Fichero {
    public static void main(String[] args) {
        String nombreArchivo = "Deathbat0.txt";

        // Crear el archivo
        crearArchivo(nombreArchivo);

        // Imprimir el contenido del archivo
        imprimirContenido(nombreArchivo);

        // Borrar el archivo
        borrarArchivo(nombreArchivo);
    }

    public static void crearArchivo(String nombreArchivo) {
        try {
            FileWriter archivo = new FileWriter(nombreArchivo);
            archivo.write("Nombre: Daniel\n");
            archivo.write("Edad: 25\n");
            archivo.write("Lenguaje de programación favorito: Java\n");
            archivo.close();
            System.out.println("El archivo " + nombreArchivo + " ha sido creado.");
        } catch (IOException e) {
            System.out.println("Ocurrió un error al crear el archivo: " + e.getMessage());
        }
    }

    public static void imprimirContenido(String nombreArchivo) {
        try {
            BufferedReader archivo = new BufferedReader(new FileReader(nombreArchivo));
            System.out.println("Contenido del archivo:");
            String linea;
            while ((linea = archivo.readLine()) != null) {
                System.out.println(linea);
            }
            archivo.close();
        } catch (IOException e) {
            System.out.println("Ocurrió un error al leer el archivo: " + e.getMessage());
        }
    }

    public static void borrarArchivo(String nombreArchivo) {
        File archivo = new File(nombreArchivo);
        if (archivo.exists()) {
            if (archivo.delete()) {
                System.out.println("El archivo " + nombreArchivo + " ha sido borrado.");
            } else {
                System.out.println("No se pudo borrar el archivo " + nombreArchivo + ".");
            }
        } else {
            System.out.println("El archivo " + nombreArchivo + " no existe.");
        }
    }
}

File: Reto # - 11/Python/Fichero.py
## DeathbatO

import os

def crear_archivo(nombre_archivo):
    with open(nombre_archivo, 'w') as archivo:
        archivo.write("Nombre: Daniel\n")
        archivo.write("Edad: 25\n")
        archivo.write("Lenguaje de programación favorito: Python\n")

def imprimir_contenido(nombre_archivo):
    with open(nombre_archivo, 'r') as archivo:
        contenido = archivo.read()
        print("Contenido del archivo:")
        print(contenido)

def borrar_archivo(nombre_archivo):
    if os.path.exists(nombre_archivo):
        os.remove(nombre_archivo)
        print(f"El archivo {nombre_archivo} ha sido borrado.")
    else:
        print(f"El archivo {nombre_archivo} no existe.")

# Nombre de usuario de GitHub para el nombre del archivo
nombre_archivo = "Deathbat0.txt"

# Crear el archivo
crear_archivo(nombre_archivo)

# Imprimir el contenido del archivo
imprimir_contenido(nombre_archivo)

# Borrar el archivo
borrar_archivo(nombre_archivo)
